<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>7 Data in Other Repositories | Best Practices for Data Package Design</title>
  <meta name="description" content="Considerations for a well designed data package including special cases based on data type, format, or acquisition method. Examples are images, documents, raw data stored in other repositories." />
  <meta name="generator" content="bookdown 0.19 and GitBook 2.6.7" />

  <meta property="og:title" content="7 Data in Other Repositories | Best Practices for Data Package Design" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Considerations for a well designed data package including special cases based on data type, format, or acquisition method. Examples are images, documents, raw data stored in other repositories." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="7 Data in Other Repositories | Best Practices for Data Package Design" />
  
  <meta name="twitter:description" content="Considerations for a well designed data package including special cases based on data type, format, or acquisition method. Examples are images, documents, raw data stored in other repositories." />
  

<meta name="author" content="Environmental Data Initiative" />


<meta name="date" content="2020-06-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="large-data-sets.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><h4><a class="brand" href="/data-package-best-practices/">Data Package Best Practices</a></h4></li>
<ul>
  <li><a href="/data-package-best-practices/EMLmetadata/">EML Metadata</a></li>
  <li><a href="/data-package-best-practices/datapackage-design/">Data Package Design</a></li>
  <li><a href="/data-package-best-practices/scientific_domain/">Data Products for Synthesis Research</a></li>
</ul> 

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Best Practice Recommendations for Data Package Design</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#data-file-type"><i class="fa fa-check"></i>Data File Type</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#entities-to-include-in-a-data-package"><i class="fa fa-check"></i>Entities to Include in a Data Package</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#splitting-and-lumping-of-data"><i class="fa fa-check"></i>Splitting and Lumping of Data</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#data-file-format"><i class="fa fa-check"></i>Data File Format</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#data-in-other-repositories"><i class="fa fa-check"></i>Data in Other Repositories</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#very-large-datasets"><i class="fa fa-check"></i>Very Large Datasets</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="key-value-pair-tables.html"><a href="key-value-pair-tables.html"><i class="fa fa-check"></i><b>1</b> Key-Value Pair Tables</a><ul>
<li class="chapter" data-level="1.1" data-path="key-value-pair-tables.html"><a href="key-value-pair-tables.html#introduction"><i class="fa fa-check"></i><b>1.1</b> Introduction</a></li>
<li class="chapter" data-level="1.2" data-path="key-value-pair-tables.html"><a href="key-value-pair-tables.html#recommendations-for-data-packages"><i class="fa fa-check"></i><b>1.2</b> Recommendations for data packages</a><ul>
<li class="chapter" data-level="1.2.1" data-path="key-value-pair-tables.html"><a href="key-value-pair-tables.html#biodiversity-data"><i class="fa fa-check"></i><b>1.2.1</b> Biodiversity data</a></li>
<li class="chapter" data-level="1.2.2" data-path="key-value-pair-tables.html"><a href="key-value-pair-tables.html#sensor-data"><i class="fa fa-check"></i><b>1.2.2</b> Sensor data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="code-in-edi.html"><a href="code-in-edi.html"><i class="fa fa-check"></i><b>2</b> Code in EDI</a><ul>
<li class="chapter" data-level="2.1" data-path="code-in-edi.html"><a href="code-in-edi.html#introduction-1"><i class="fa fa-check"></i><b>2.1</b> Introduction</a><ul>
<li class="chapter" data-level="2.1.1" data-path="code-in-edi.html"><a href="code-in-edi.html#background"><i class="fa fa-check"></i><b>2.1.1</b> Background</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="code-in-edi.html"><a href="code-in-edi.html#recommendations-for-data-packages-1"><i class="fa fa-check"></i><b>2.2</b> Recommendations for data packages</a><ul>
<li class="chapter" data-level="2.2.1" data-path="code-in-edi.html"><a href="code-in-edi.html#softwarecode-in-edi"><i class="fa fa-check"></i><b>2.2.1</b> Software/code in EDI</a></li>
<li class="chapter" data-level="2.2.2" data-path="code-in-edi.html"><a href="code-in-edi.html#documenting-softwarecode"><i class="fa fa-check"></i><b>2.2.2</b> Documenting software/code</a></li>
<li class="chapter" data-level="2.2.3" data-path="code-in-edi.html"><a href="code-in-edi.html#linking-between-code-and-data"><i class="fa fa-check"></i><b>2.2.3</b> Linking between code and data</a></li>
<li class="chapter" data-level="2.2.4" data-path="code-in-edi.html"><a href="code-in-edi.html#external-software"><i class="fa fa-check"></i><b>2.2.4</b> External software</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="code-in-edi.html"><a href="code-in-edi.html#resources"><i class="fa fa-check"></i><b>2.3</b> Resources</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="model-based-datasets.html"><a href="model-based-datasets.html"><i class="fa fa-check"></i><b>3</b> Model Based Datasets</a><ul>
<li class="chapter" data-level="3.1" data-path="model-based-datasets.html"><a href="model-based-datasets.html#introduction-2"><i class="fa fa-check"></i><b>3.1</b> Introduction</a><ul>
<li class="chapter" data-level="3.1.1" data-path="model-based-datasets.html"><a href="model-based-datasets.html#background-1"><i class="fa fa-check"></i><b>3.1.1</b> Background</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="model-based-datasets.html"><a href="model-based-datasets.html#recommendations-for-data-packages-2"><i class="fa fa-check"></i><b>3.2</b> Recommendations for data packages</a><ul>
<li class="chapter" data-level="3.2.1" data-path="model-based-datasets.html"><a href="model-based-datasets.html#referencing-models-in-eml"><i class="fa fa-check"></i><b>3.2.1</b> Referencing models in EML</a></li>
<li class="chapter" data-level="3.2.2" data-path="model-based-datasets.html"><a href="model-based-datasets.html#model-code"><i class="fa fa-check"></i><b>3.2.2</b> Model code</a></li>
<li class="chapter" data-level="3.2.3" data-path="model-based-datasets.html"><a href="model-based-datasets.html#model-input-and-output-data"><i class="fa fa-check"></i><b>3.2.3</b> Model input and output data</a></li>
<li class="chapter" data-level="3.2.4" data-path="model-based-datasets.html"><a href="model-based-datasets.html#model-parameters"><i class="fa fa-check"></i><b>3.2.4</b> Model parameters</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="model-based-datasets.html"><a href="model-based-datasets.html#resources-1"><i class="fa fa-check"></i><b>3.3</b> Resources</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="still-images.html"><a href="still-images.html"><i class="fa fa-check"></i><b>4</b> Still Images</a><ul>
<li class="chapter" data-level="4.1" data-path="key-value-pair-tables.html"><a href="key-value-pair-tables.html#introduction"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="key-value-pair-tables.html"><a href="key-value-pair-tables.html#recommendations-for-data-packages"><i class="fa fa-check"></i><b>4.2</b> Recommendations for data packages</a><ul>
<li class="chapter" data-level="4.2.1" data-path="still-images.html"><a href="still-images.html#recommendations-for-metadata"><i class="fa fa-check"></i><b>4.2.1</b> Recommendations for Metadata</a></li>
<li class="chapter" data-level="4.2.2" data-path="still-images.html"><a href="still-images.html#image-catalog"><i class="fa fa-check"></i><b>4.2.2</b> Image Catalog</a></li>
<li class="chapter" data-level="4.2.3" data-path="still-images.html"><a href="still-images.html#data-extracted-from-images"><i class="fa fa-check"></i><b>4.2.3</b> Data Extracted from Images</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="still-images.html"><a href="still-images.html#resources-2"><i class="fa fa-check"></i><b>4.3</b> Resources</a><ul>
<li class="chapter" data-level="4.3.1" data-path="still-images.html"><a href="still-images.html#examples-in-edi"><i class="fa fa-check"></i><b>4.3.1</b> Examples in EDI</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html"><i class="fa fa-check"></i><b>5</b> Genomic and Metagenomic Data</a><ul>
<li class="chapter" data-level="5.1" data-path="key-value-pair-tables.html"><a href="key-value-pair-tables.html#introduction"><i class="fa fa-check"></i><b>5.1</b> Introduction</a><ul>
<li class="chapter" data-level="5.1.1" data-path="code-in-edi.html"><a href="code-in-edi.html#background"><i class="fa fa-check"></i><b>5.1.1</b> Background</a></li>
<li class="chapter" data-level="5.1.2" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#types-of-genomics-datasets-commonly-encountered"><i class="fa fa-check"></i><b>5.1.2</b> Types of Genomics Datasets Commonly Encountered</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#recommendations-for-datasets"><i class="fa fa-check"></i><b>5.2</b> Recommendations for Datasets</a><ul>
<li class="chapter" data-level="5.2.1" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#metadata-for-all-genomics-datasets-archived-in-edi"><i class="fa fa-check"></i><b>5.2.1</b> Metadata for All Genomics Datasets Archived in EDI</a></li>
<li class="chapter" data-level="5.2.2" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#genomics-data-summary"><i class="fa fa-check"></i><b>5.2.2</b> Genomics Data Summary</a></li>
<li class="chapter" data-level="5.2.3" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#genomics-data-with-ancillary-data"><i class="fa fa-check"></i><b>5.2.3</b> Genomics Data with Ancillary Data</a></li>
<li class="chapter" data-level="5.2.4" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#derived-data-products"><i class="fa fa-check"></i><b>5.2.4</b> Derived data products</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="code-in-edi.html"><a href="code-in-edi.html#resources"><i class="fa fa-check"></i><b>5.3</b> Resources</a><ul>
<li class="chapter" data-level="5.3.1" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#tips-for-locating-metadata"><i class="fa fa-check"></i><b>5.3.1</b> Tips for locating metadata</a></li>
<li class="chapter" data-level="5.3.2" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#links-to-various-repositories"><i class="fa fa-check"></i><b>5.3.2</b> Links to various repositories</a></li>
<li class="chapter" data-level="5.3.3" data-path="genomic-and-metagenomic-data.html"><a href="genomic-and-metagenomic-data.html#related-best-practices-and-templates"><i class="fa fa-check"></i><b>5.3.3</b> Related Best Practices and Templates</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="large-data-sets.html"><a href="large-data-sets.html"><i class="fa fa-check"></i><b>6</b> Large Data Sets</a><ul>
<li class="chapter" data-level="6.1" data-path="large-data-sets.html"><a href="large-data-sets.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a><ul>
<li class="chapter" data-level="6.1.1" data-path="large-data-sets.html"><a href="large-data-sets.html#background-2"><i class="fa fa-check"></i><b>6.1.1</b> Background</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="large-data-sets.html"><a href="large-data-sets.html#recommendations-for-data-packages-3"><i class="fa fa-check"></i><b>6.2</b> Recommendations for data packages</a><ul>
<li class="chapter" data-level="6.2.1" data-path="large-data-sets.html"><a href="large-data-sets.html#physical-storage"><i class="fa fa-check"></i><b>6.2.1</b> Physical Storage</a></li>
<li class="chapter" data-level="6.2.2" data-path="large-data-sets.html"><a href="large-data-sets.html#data-package"><i class="fa fa-check"></i><b>6.2.2</b> Data package</a></li>
<li class="chapter" data-level="6.2.3" data-path="large-data-sets.html"><a href="large-data-sets.html#package-metadata"><i class="fa fa-check"></i><b>6.2.3</b> Package Metadata</a></li>
<li class="chapter" data-level="6.2.4" data-path="large-data-sets.html"><a href="large-data-sets.html#eml"><i class="fa fa-check"></i><b>6.2.4</b> EML</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="large-data-sets.html"><a href="large-data-sets.html#references"><i class="fa fa-check"></i><b>6.3</b> References</a><ul>
<li class="chapter" data-level="6.3.1" data-path="large-data-sets.html"><a href="large-data-sets.html#eml-documentation"><i class="fa fa-check"></i><b>6.3.1</b> EML documentation</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="large-data-sets.html"><a href="large-data-sets.html#potential-issues"><i class="fa fa-check"></i><b>6.4</b> Potential Issues</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html"><i class="fa fa-check"></i><b>7</b> Data in Other Repositories</a><ul>
<li class="chapter" data-level="7.1" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html#considerations-for-creating-linked-repository-records"><i class="fa fa-check"></i><b>7.2</b> Considerations for creating linked repository records</a><ul>
<li class="chapter" data-level="7.2.1" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html#cases-supporting-linked-datasets"><i class="fa fa-check"></i><b>7.2.1</b> Cases supporting linked datasets</a></li>
<li class="chapter" data-level="7.2.2" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html#when-to-avoid-linked-datasets"><i class="fa fa-check"></i><b>7.2.2</b> When to avoid linked datasets</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html#recommendations-for-data-packages-4"><i class="fa fa-check"></i><b>7.3</b> Recommendations for Data Packages</a><ul>
<li class="chapter" data-level="7.3.1" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html#package-metadata-1"><i class="fa fa-check"></i><b>7.3.1</b> Package Metadata</a></li>
<li class="chapter" data-level="7.3.2" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html#contents-of-the-manifest-datatable-entity"><i class="fa fa-check"></i><b>7.3.2</b> Contents of the manifest dataTable entity</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="data-in-other-repositories-1.html"><a href="data-in-other-repositories-1.html#example-data-packages"><i class="fa fa-check"></i><b>7.4</b> Example data packages</a></li>
</ul></li>
<li class="divider"></li>
<ul>
    <li><h4><a href="https://EnvironmentalDataInitiative.org" target="blank">Environmental Data Initiative</a></h4></li>
    <li><img class="img-responsive center-block" src="../../data-package-best-practices/images/EDI-logo-svg-240.png" alt="EDI Logo" height="90" width="90"/><li>
</ul>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Best Practices for Data Package Design</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="data-in-other-repositories-1" class="section level1">
<h1><span class="header-section-number">7</span> Data in Other Repositories</h1>
<p>Contributors: Greg Maurer, Margaret O’Brien, Corinna Gries, Tim Whiteaker, Sarah Elmendorf</p>
<div id="introduction-4" class="section level2">
<h2><span class="header-section-number">7.1</span> Introduction</h2>
<p>A wide variety of data repositories are available to ecologists for publishing their data. The choice of where to publish a dataset is determined by many factors. The funding agency or a journal may require a certain repository (e.g., NSF BCO-DMO, NSF ADC, USDA ADC, DOE ESS-DIVE); the research subject may be best supported by a very specialized repository (e.g., AmeriFlux, GenBank); or the data were originally submitted to a general purpose repository with minimal metadata requirements (e.g., DRYAD, Figshare, Zenodo). To enhance data discoverability and interoperability, users and data managers may find it helpful to create additional repository records that are linked to these published datasets.</p>
<p>These concepts must be understood to clarify this best practices document:</p>
<ul>
<li><p>A <strong>dataset</strong> consists of a data file, or files, containing named variables (say “Soil_T_10cm”) with assigned values (the temperature numbers themselves) AND the accompanying metadata describing attributes of the data and other information.</p></li>
<li><p>A <strong>duplicate dataset</strong> contains data and/or metadata that are identical to those in another public data repository.</p></li>
<li><p>A <strong>linked dataset</strong> contains metadata, but the only data present are links to the unique identifier(s) of data held in another repository.</p></li>
</ul>
<p>For the purposes of this document, all three dataset types above are published in online research data repositories and are assigned unique digital identifiers (DOIs).</p>
<p>This document discusses reasons for creating additional repository records for a published dataset, i.e., linked datasets pointing to other repositories, and recommendations of how to develop such records. It was written for the intended audience of a Long Term Ecological Research (LTER) information manager (IM), though it is applicable to anyone handling data in the context of a larger ecological research program. EDI is used as the representative data repository in this document. The same practices could be applied to other similar repositories.</p>
</div>
<div id="considerations-for-creating-linked-repository-records" class="section level2">
<h2><span class="header-section-number">7.2</span> Considerations for creating linked repository records</h2>
<p>Data managers and users have several reasons to consider creating linked datasets, i.e. additional repository records linked to a published dataset. For the LTER IM, this practice involves creating a new data package in the EDI repository that links to data already published in another (non-EDI) repository. However, care should be taken to <strong>avoid duplicating the same data in multiple repositories</strong>. The practice of duplicating research data in multiple repositories is discouraged because a) this creates a burden of maintaining multiple datasets to avoid divergence between them, and b) it creates confusion for data re-users who may be led to download the same data multiple times.</p>
<p>There are clear cases in which creating linked datasets offers benefits to data managers and users. There are also cases in which this practice should be avoided. Both types of cases are described in the sections below, but there are situations in which the pros and cons of this practice will need to be weighed. Experience with data user behavior and support is paramount for such decisions and may be made differently by different research groups.</p>
<div id="cases-supporting-linked-datasets" class="section level3">
<h3><span class="header-section-number">7.2.1</span> Cases supporting linked datasets</h3>
<ul>
<li><p><strong>Requirements dictate multiple repositories:</strong> Large research projects or sites are frequently funded by different agencies and programs. Data collection may be supported by several such funding streams and, hence, fall in the purview of more than one requirement for data being archived in a particular repository. In some cases, data repositories already accommodate such requirements by linking or replicating data appropriately. Best examples are LTER data in EDI, NSF BCO-DMO and NSF ADC.</p></li>
<li><p><strong>Improved discoverability:</strong> Data from a large research project or sites become better discoverable when published and, hence, discoverable within the same repository. This practice also puts the data into the larger context of all research and ancillary data at that location aiding in interpretation and analysis. It, furthermore, allows the site to easily build a more complete data catalog for reporting or other administrative purposes without having to pull data citations from different sources. By the same token, the same type of data from disparate projects or sites may be more discoverable and easier to integrate when stored in a specialized repository focused on a particular type of data. For instance, specialized data repositories can enforce data formatting or quality standards and require metadata elements that enhance search, discovery and reuse of particular types of data across projects in a way that is not possible using a generalized metadata form (EML) and repository (EDI).</p></li>
<li><p><strong>Adding important metadata</strong>: If data were originally submitted to a general purpose repository with minimal metadata requirements (e.g., Dublin core) additional metadata (e.g., EML) may be needed for discoverability, reusability and integration. By creating a new repository record linked to the published dataset, richer and more useful metadata can be added and utilized.</p></li>
<li><p><strong>Derived data products:</strong> In many cases raw data are best handled in a specialized repository, but derived data products based on those raw data are, or should be, published in a different repository. For example, genomics data such as raw sequencing data are best archived in a specialized repository like <a href="https://www.ncbi.nlm.nih.gov/"><span class="underline">NCBI GenBank</span></a>, but derived products such as trait data or diversity metrics would be most useful in EDI (see <a href="https://ediorg.github.io/data-package-best-practices/datapackage-design/genomic-and-metagenomic-data.html"><span class="underline">the Genomics best practices document</span></a> for more). The same principles apply whenever a project or researcher has developed specialized procedures to process raw data or compute aggregations that are more appropriate for most users. In these cases it is important to maintain links between the derived data and the raw data when they are in separate repositories.</p></li>
</ul>
</div>
<div id="when-to-avoid-linked-datasets" class="section level3">
<h3><span class="header-section-number">7.2.2</span> When to avoid linked datasets</h3>
<ul>
<li><p><strong>Site- or Project-relevant data from outside research groups or agencies:</strong> Although it may help with some aspects of data discovery it is generally not recommended to create linked datasets (or duplicates) for data collected and managed by entirely different research groups or agencies. <strong><em>In these cases, however, it is recommended to place a pointer to such repositories on a project website.</em></strong></p></li>
<li><p><strong>Linking between repositories that are DataONE member nodes</strong>: The DataONE aggregator provides access to earth data held in numerous repositories through a unified interface. It is not recommended to duplicate this functionality by creating additional links between repositories.</p></li>
</ul>
</div>
</div>
<div id="recommendations-for-data-packages-4" class="section level2">
<h2><span class="header-section-number">7.3</span> Recommendations for Data Packages</h2>
<p>When the decision is made that a dataset needs to be accessible in more than one repository the following considerations should guide the process:</p>
<ul>
<li><p>The metadata need to clearly state the original location of the data</p></li>
<li><p>Assigning more than one unique identifier (e.g., DOI) to the same data can create confusion when data is cited. Thus, the DOI of the data itself should be clearly stated in the linked data package.</p></li>
<li><p>The original repository should receive appropriate credit</p></li>
<li><p>One data package that functions as an inventory of data in other repositories for a site is an option, but will not improve discoverability of single datasets.</p></li>
</ul>
<p>In EDI, the linked data package can be assembled using standard practices and EML metadata elements, but the included data entities must clearly lead the data user to files held in outside repositories. In addition, the package metadata should communicate the essential elements needed for data discovery (subject matter, authors, location, time-frame, etc.) and a brief description of how the data may be accessed and re-used via the outside repository. Metadata elements to be created in EML, and contents of the included data entities are described below.</p>
<div id="package-metadata-1" class="section level3">
<h3><span class="header-section-number">7.3.1</span> Package Metadata</h3>
<p>The EML metadata that will accompany the data package in EDI must contain the essential information for users to discover and use the data at the outside data repositories.</p>
<ul>
<li><p><strong>Abstract:</strong> Describe the collection generally. Since the actual data are held elsewhere, be sure to describe the location of the data being linked to. Also provide sufficient information for users to find and re-use the data entities at the outside data repository.</p></li>
<li><p><strong>Creators</strong></p></li>
<li><p><strong>Contacts</strong></p></li>
<li><p><strong>Methods:</strong> Collection/generation methods for the data entities at the linked repository. If the methods are well-described at the outside repository being linked to, this element can simply refer users there. In addition, instructions for data access at the linked data repository should be provided.</p></li>
<li><p><strong>Geographic description</strong> and <strong>coordinates</strong>: At a minimum these elements should define a bounding box that will make the data package discoverable through EDI, DataOne, or other geographic search interfaces. Additional coordinates can be given in the manifest entity below.</p></li>
<li><p><strong>Keywords:</strong> If this is an inventory rather than a single dataset duplication held at a different repository, include the keyword &quot;data inventory&quot; and thematic keywords that describe the data entities in the other repository.</p></li>
<li><p><strong>Data Entity:</strong> This is most likely a simple tabular manifest (inventory of holdings) that describes the data entities held in another data repository (or repositories). This should be a <a href="https://eml.ecoinformatics.org/schema/eml-dataTable_xsd.html#eml-dataTable.xsd"><span class="underline">dataTable</span></a> entity, described as you would for any dataTable with regular column descriptions. In rare cases the data themselves may be duplicated, though this is only recommended if additional metadata must be added to interpret or enrich the data.</p></li>
</ul>
</div>
<div id="contents-of-the-manifest-datatable-entity" class="section level3">
<h3><span class="header-section-number">7.3.2</span> Contents of the manifest dataTable entity</h3>
<ul>
<li><p>Any data package that links to data held in an outside repository should contain at least one data entity: a manifest (inventory) of data holdings at a linked repository(ies). The manifest should be written in a tabular data format (such as a .csv) that can be described as a dataTable entity in EML. For the specific case of genomics data see <a href="https://ediorg.github.io/data-package-best-practices/datapackage-design/genomic-and-metagenomic-data.html"><span class="underline">here</span></a>.</p></li>
<li><p>For each outside data holding, the manifest (dataTable entity) should have a row that includes persistent links such as DOIs to any data entities at the linked repository, relevant package identifiers, and descriptors for the data entity. The complete content of the manifest (inventory of holdings) would be dictated by the structure of the linked repository and the type of data entities there. Suggested columns are:</p></li>
</ul>
<p><strong>Table 1:</strong> Required columns in the manifest entity.</p>
<table>
<thead>
<tr class="header">
<th align="left">Column name</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Filename(s)</td>
<td align="left">Filename at the repository being linked to</td>
</tr>
<tr class="even">
<td align="left">Format</td>
<td align="left">File format of above</td>
</tr>
<tr class="odd">
<td align="left">Title/description</td>
<td align="left">Title and/or brief description of the data resource</td>
</tr>
<tr class="even">
<td align="left">External package/entity ID</td>
<td align="left">Unique identifier at the repository being linked to.</td>
</tr>
<tr class="odd">
<td align="left">Publication_DOI</td>
<td align="left">DOI of of data published at the repository being linked to</td>
</tr>
<tr class="even">
<td align="left">Repository_URL</td>
<td align="left">URL of the repository being linked to</td>
</tr>
</tbody>
</table>
<p><strong>Table 2:</strong> Optional columns in the manifest entity. These may be included in the manifest when needed to describe multiple data entities.</p>
<table>
<thead>
<tr class="header">
<th align="left">Column name</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Latitude</td>
<td align="left">Latitude in decimal degrees, WGS84 datum, associated with the data resource. For bounding boxes, use North_latitude and South_latitude columns.</td>
</tr>
<tr class="even">
<td align="left">Longitude</td>
<td align="left">Longitude in decimal degrees, WGS84 datum, associated with the data resource. For bounding boxes, use East_longitude and West_longitude columns.</td>
</tr>
<tr class="odd">
<td align="left">Start_datetime</td>
<td align="left">Starting datetime of the linked data file</td>
</tr>
<tr class="even">
<td align="left">End_datetime</td>
<td align="left">Ending datetime of the linked data file (NA if data collection is ongoing)</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="example-data-packages" class="section level2">
<h2><span class="header-section-number">7.4</span> Example data packages</h2>
<p>Each of the EDI packages below contain dataTable entities with links to outside repositories. These dataTables serve as the manifests (inventory) for the externally-held data described in the rest of the metadata.</p>
<p><strong>Table 3.</strong> Linked-repository packages at EDI</p>
<table>
<thead>
<tr class="header">
<th align="left">Title</th>
<th align="left">Description</th>
<th align="left">EDI packageID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Mass and energy fluxes from the US-Jo2 AmeriFlux eddy covariance tower in Tromble Weir experimental watershed at the Jornada Basin LTER site, 2010-ongoing</td>
<td align="left">This data package links to eddy covariance data from a Jornada Basin LTER tower. The data are held at the AmeriFlux data repository (<a href="https://ameriflux.lbl.gov"><span class="underline">https://ameriflux.lbl.gov</span></a>)</td>
<td align="left"><a href="https://portal.edirepository.org/nis/mapbrowse?scope=knb-lter-jrn&amp;identifier=210338005"><span class="underline">knb-lter-jrn.210338005</span></a></td>
</tr>
</tbody>
</table>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="large-data-sets.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
